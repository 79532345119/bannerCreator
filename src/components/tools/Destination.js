import React from "react"


export const Destination = (props ) => {
    return ( <div className="tools_elem" >
        <label htmlFor="destLink">ссылка, куда ведет баннер</label>
        <input type="text" id="destLink" value={props.destLink} onChange={e=>props.setDestLink(e.target.value)} />
    </div>      
    )
}
