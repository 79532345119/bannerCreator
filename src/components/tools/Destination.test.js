import React from "react"
import {create} from "react-test-renderer"
import {Destination} from "./Destination"


describe("Destination component", ()=> {
    test("Destination component renders correctly", ()=> {
        const component = create(<Destination props="link"/>);
        expect(component).toMatchSnapshot();
    })
})

