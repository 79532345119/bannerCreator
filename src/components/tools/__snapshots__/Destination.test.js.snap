// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Destination component Destination component renders correctly 1`] = `
<div
  className="tools_elem"
>
  <label
    htmlFor="destLink"
  >
    ссылка, куда ведет баннер
  </label>
  <input
    id="destLink"
    onChange={[Function]}
    type="text"
  />
</div>
`;
